cg/meta/workflow/fluffy.py

```diff
@@ -0,0 +1,243 @@
+from pathlib import Path
+import logging
+import csv
+from subprocess import CalledProcessError
+from typing import Any, Optional
+
+import shutil
+import datetime as dt
+
+from cg.apps.hermes.hermes_api import HermesApi
+from cg.exc import BundleAlreadyAddedError
+from cg.utils import Process
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.apps.tb import TrailblazerAPI
+from cg.apps.lims import LimsAPI
+from cg.store import Store, models
+import os
+from cg.constants import Pipeline, CASE_ACTIONS
+
+LOG = logging.getLogger(__name__)
+
+
+class FluffyAnalysisAPI:
+    def __init__(
+        self,
+        housekeeper_api: HousekeeperAPI,
+        trailblazer_api: TrailblazerAPI,
+        hermes_api: HermesApi,
+        lims_api: LimsAPI,
+        status_db: Store,
+        config: dict,
+    ):
+        self.housekeeper_api = housekeeper_api
+        self.trailblazer_api = trailblazer_api
+        self.status_db = status_db
+        self.lims_api = lims_api
+        self.hermes_api = hermes_api
+        self.root_dir = Path(config["root_dir"])
+        self.process = Process(binary=config["binary_path"])
+        self.fluffy_config = Path(config["config_path"])
+
+    def get_samplesheet_path(self, case_id: str) -> Path:
+        return Path(self.root_dir, case_id, "SampleSheet.csv")
+
+    def get_workdir_path(self, case_id: str) -> Path:
+        return Path(self.root_dir, case_id, "fastq")
+
+    def get_fastq_path(self, case_id: str, sample_id: str) -> Path:
+        return Path(self.root_dir, case_id, "fastq", sample_id)
```

Comment:
```suggestion
        return self.get_workdir_path(case_id) / sample_id
```

Reply:
I slacked about this, dont really want to change this...